packages/tester-node/src/mapi/consolidated-query.ts

```diff
@@ -11,21 +11,25 @@ const patientId = getEnvVarOrFail("PATIENT_ID");
 const sleep = (timeInMs: number) => new Promise(resolve => setTimeout(resolve, timeInMs));
 
 async function main() {
-  console.log(`Calling startConsolidatedQuery...`);
-
   const metriport = new MetriportMedicalApi(apiToken, {
     baseAddress: apiUrl,
     timeout: 60_000,
   });
 
-  const deprecated = await metriport.getPatientConsolidated(
-    patientId,
-    ["DocumentReference", "Appointment"],
-    "2021-03-01",
-    "2023-04-23"
-  );
-  console.log(`Result: ${JSON.stringify(deprecated, null, 2)}`);
+  // console.log(`Calling getPatientConsolidated...`);
```

Comment:
nit: just delete entirely?

Reply:
Good point. I commented out b/c it takes long to process and its deprecated, but then yeah I should remove it - or just leave as is, we should prob move this to `packages/tester-node` soon.